docs/guide/blog.md

```diff
@@ -272,7 +301,10 @@ By following these steps, you have implemented all of the code required to creat
 
 Try it out! If the chain is yet not started, run `starport chain serve`.
 
-Create a post:
+__Note__: Please make sure you have `$GOPATH$/bin` in your PATH.
```

Comment:
I think we don't need to cover this part here again. It's already a part of the [Install Starport](https://docs.starport.network/guide/install.html) guideline which where added a link to: https://golang.org/doc/gopath_code#GOPATH